src/v2/ion/RemediationEnum.js

```diff
@@ -0,0 +1,42 @@
+const FORMS = {
+  IDENTIFY: 'identify',
+  SELECT_IDENTIFY: 'select-identify',
+  SELECT_ENROLL_PROFILE: 'select-enroll-profile',
+  ENROLL_PROFILE: 'enroll-profile',
+
+  SELECT_AUTHENTICATOR_AUTHENTICATE: 'select-authenticator-authenticate',
+  AUTHENTICATOR_VERIFICATION_DATA: 'authenticator-verification-data',
+  CHALLENGE_AUTHENTICATOR: 'challenge-authenticator',
+
+  SELECT_AUTHENTICATOR_ENROLL: 'select-authenticator-enroll',
+  SELECT_AUTHENTICATOR_ENROLL_DATA: 'select-authenticator-enroll-data',
+  AUTHENTICATOR_ENROLLMENT_DATA: 'authenticator-enrollment-data',
+  ENROLL_AUTHENTICATOR: 'enroll-authenticator',
+  SKIP: 'skip',
+
+  SUCCESS_REDIRECT: 'success-redirect',
+  REDIRECT_IDP: 'redirect-idp',
+
+  DEVICE_CHALLENGE_POLL: 'device-challenge-poll',
+  DEVICE_APPLE_SSO_EXTENSION: 'device-apple-sso-extension',
+  CANCEL_TRANSACTION: 'cancel-transaction',
+  LAUNCH_AUTHENTICATOR: 'launch-authenticator',
+};
+
+const NEED_HIDE_CANCEL_FORMS = [
+  FORMS.IDENTIFY,
+  FORMS.SUCCESS_REDIRECT,
+  FORMS.ENROLL_PROFILE,
+  FORMS.SELECT_AUTHENTICATOR_ENROLL,
+  FORMS.AUTHENTICATOR_ENROLLMENT_DATA,
+  FORMS.ENROLL_AUTHENTICATOR,
+  FORMS.REDIRECT_IDP,
+  // TODO: remove following when deprecating `factor`
+  'select-factor-enroll',
+  'enroll-factor'
+];
+
+export default {
```

Comment:
Instead of exporting a whole objects individual exports are better since that supports Webpack tree-shaking (if we are/were to use in future). Something like this - https://stackoverflow.com/questions/56816190/what-is-the-right-way-for-better-treeshaking-in-webpack

Reply:
- I'd like to keep under a namespace like `FORMS`. it feels a bit more readable IMO unless to prefix name with something like `ION_FORM`.?
- All of name would/shall be used. Also this file is small enough to be concerned during tree shaking.